litmus-portal/frontend/src/pages/WorkflowInfoStats/InfoSection/index.tsx

```diff
@@ -0,0 +1,171 @@
+            )}
+            {data && (
+              <Typography>
+                Namespace :{' '}
+                <span className={classes.infoHint}>
+                  {
+                    YAML.parse(
+                      data?.ListWorkflow.workflows[0].workflow_manifest
+                    ).metadata.namespace
+                  }
+                </span>
+              </Typography>
+            )}
+          </div>
+
+          {/* Column 2 */}
+          <div>
+            <Typography className={classes.infoHeader}>Agent :</Typography>
+            <Typography>
+              Name :{' '}
+              <span className={classes.infoHint}>
+                {data.ListWorkflow.workflows[0].cluster_name}
+              </span>
+            </Typography>
+            <Typography>
+              Id :{' '}
+              <span className={classes.infoHint}>
+                {data.ListWorkflow.workflows[0].cluster_id}
+              </span>
+            </Typography>
+          </div>
+          {/* Column 3 */}
+          <div>
+            <Typography className={classes.infoHeader}>
+              Total Runs : {/* Replace with LIST_API */}
+              {data.ListWorkflow.workflows[0].workflow_runs?.length}
+            </Typography>
+            <Typography>
+              Last Run :{' '}
+              <span className={classes.infoHint}>
+                {timeDifferenceForDate(
+                  data.ListWorkflow.workflows[0].updated_at
+                )}
+              </span>
+            </Typography>
+            <Typography>
+              Next Run :{' '}
+              {data.ListWorkflow.workflows[0].cronSyntax !== undefined && (
+                <span className={classes.infoHint}>
+                  {parser
+                    .parseExpression(data.ListWorkflow.workflows[0].cronSyntax)
+                    .next()
+                    .toString()}
+                </span>
+              )}
+            </Typography>
+          </div>
+          {/* Column 4 */}
+          <div className={classes.regularity}>
+            <Typography className={classes.infoHeader}>Regularity :</Typography>
+            {data.ListWorkflow.workflows[0].cronSyntax === '' ? (
+              <Typography>Non cron workflow</Typography>
+            ) : (
+              data.ListWorkflow.workflows[0].cronSyntax !== undefined && (
+                <Typography>
+                  {cronstrue.toString(
+                    data.ListWorkflow.workflows[0].cronSyntax
+                  )}
+                </Typography>
+              )
+            )}
+          </div>
+        </div>
+        {showMore ? (
+          <WorkflowStats
+            workflowID={data.ListWorkflow.workflows[0].workflow_id}
+            isCron={data.ListWorkflow.workflows[0].cronSyntax !== ''}
+            noOfWorkflowRuns={
+              data.ListWorkflow.workflows[0].workflow_runs?.length ?? 0
+            }
+          />
+        ) : (
+          <></>
+        )}
```

Comment:
Instead of using `showMore ? <component> : <null> ` we may use `showMore && <component>` so the we do not have to give empty tags